EvalData/models/base_models.py

```diff
@@ -645,6 +646,12 @@ class TextPair(EvalItem):
         verbose_name=_('Target text'),
     )
 
+    # user for AI-assisted annotation
+    mqm = models.TextField(
+        blank=True,
+        verbose_name=_('MQM Annotations'),
```

Comment:
It seems that this change requires providing a default value for existing data.

Reply:
I propose to fix it as in https://github.com/AppraiseDev/Appraise/tree/romang/develop-mqm